test/smoke/appServers/global-resources/telemetryprocessors_applicationinsights.json

```diff
@@ -77,6 +86,16 @@
             }
           ],
           "id": "attributes/extract"
+        },
+        {
+          "type": "log",
+          "name": {
```

Comment:
let's change this to `body` since that's what it's called in OTel log data model